examples/coap_server/coap_server.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2015 Cesanta Software Limited
+ * All rights reserved
+ *
+ * This program listens on 5683 for CoAP messages, 
+ * sends ACK is nessesary and dump everything received.
+ * It is possible to use ../coap_client to send message.
+ */
+
+#include "fossa.h"
+
+static char* g_default_address = "udp://5683";
```

Comment:
`udp://:5683` is the documented format, this without colon works but should be eventually removed and it shouldn't be present in our examples.